client.py

```diff
@@ -0,0 +1,16 @@
+from eva.server.db_api import connect
+import nest_asyncio
+nest_asyncio.apply()
+
+# Get Cursor
+connection = connect(host = '0.0.0.0', port = 5432) # hostname, port of the server where EVADB is running
+cursor = connection.cursor()
+
+# Drop a UDF
+cursor.execute("DROP UDF IF EXISTS FastRCNNObjectDetector")
+response = cursor.fetch_all()
+
+# Re-Create the UDF
+cmd = "CREATE UDF IF NOT EXISTS FastRCNNObjectDetector INPUT  (frame NDARRAY UINT8(3, ANYDIM, ANYDIM)) OUTPUT (labels NDARRAY STR(ANYDIM), bboxes NDARRAY FLOAT32(ANYDIM, 4), scores NDARRAY FLOAT32(ANYDIM)) TYPE Classification CATEGORY 'VehicleCount' IMPL 'eva/udfs/fastrcnn_object_detector.py';"
```

Comment:
`CATEGORY 'VehicleCount'` is the newly added meta.

If the UDF already exists, we can't add it. Wrote a simple client that would drop a UDF and add it back with Category meta